### ISSUE TITLE
QtMEL & Visual Studio 2013

### ISSUE THREAD

**Issue by jschlu** [kibsoft/QtMEL#36]
Using Qt 5.6 and Visual Studio 2013, would I need to manually build your libraries, or can I use the provided prebuilt libs?

**Comment by kibsoft** [comment]
Hello. Unfortunately, you need to build it manually.

**Comment by jschlu** [comment]
I appreciate the quick response! I am looking for a solution to:
- Record (and output) multiple live camera streams
- Display current camera feed
- Perform image subtraction to support motion detection

Would your libraries be able to handle multiple cameras at once? Also, since you have worked with OpenCV, do you know if it a simple process to convert from a QImage to an image that OpenCV can use in order to subtract images? Thank you for the assistance!

**Comment by kibsoft** [comment]
> Would your libraries be able to handle multiple cameras at once?
Yes, all you need is to create a new image grabber(by subclassing AbstractImageGrabber) or use CameraGrabber(if it suits). Then just create any quantity of grabbers you want (how many cameras you have).

> Also, since you have worked with OpenCV, do you know if it a simple process to convert from a QImage to an image that OpenCV can use in order to subtract images?
It is quite simple when we have ready solution :)
Here it is: https://umanga.wordpress.com/2010/04/19/how-to-covert-qt-qimage-into-opencv-iplimage-and-wise-versa/
